scripts/parse_ba_results.py

```diff
@@ -22,6 +25,7 @@ def main(lims, args):
     log = []
     errors = False
     count_per = "row"  # BioAnalyzer XML numbers wells row-wise
+    ngi_sample_id_pattern = r"(p|P)\d{5}_\d{3}"
```

Comment:
I guess this pattern needs to be changed because we will have project IDs with more or less than 5 digits, and samples IDs with more than 3 digits. Example: P123456_1001

Reply:
Good catch!
[2ebda4f](https://github.com/SciLifeLab/scilifelab_epps/pull/296/commits/2ebda4ff0979315d1b5663cf98789f20a3047bea)